src/bp-templates/bp-nouveau/includes/activity/ajax.php

```diff
@@ -532,6 +532,14 @@ function bp_nouveau_ajax_post_update() {
 		wp_send_json_error();
 	}
 
+	if ( ! strlen( trim( strip_tags($_POST['content'] ) ) ) ) {
+		wp_send_json_error(
+			array(
+				'message' => __( 'Please enter some content to post.', 'buddyboss' ),
+			)
+		);
+	}
+
```

Comment:
@buddy4kp You should have used the validation written below. This code check for content is empty striping tags but it will give errors even if we have content like media, link preview, document etc. we need to allow users to post only media without content. so this condition should always check with them.